src/ua.c

```diff
@@ -681,6 +681,7 @@ static int sdp_connection(struct mbuf *mb, int *af, struct sa *sa)
 void sipsess_conn_handler(const struct sip_msg *msg, void *arg)
 {
 	struct config *config = conf_config();
+	const char magic_branch[] = "z9hG4bK";
```

Comment:
Maybe:

```suggestion
	const char magic_branch[] = RE_RFC3261_BRANCH_ID;
```
and define:

```c
#define RE_RFC3261_BRANCH_ID         "z9hG4bK"
```

Reply:
Thank you! Done.